pkgs/development/python-modules/pyexpect/default.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  unittestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "pyexpect";
+  version = "1.0.22";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-ZZNR5+6JI7Qt6Kd0+r/IBqzwc3fX/Rny6kQS749hnGo=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  nativeCheckInputs = [
+    unittestCheckHook
+  ];
```

Comment:
may be able to just use pytestCheckHook here instead. Not sure.

Reply:
Package does not use pytest, it does run with pytest but 1 test passes with unittest while failing with pytest.